src/mlpack/methods/ann/activation_functions/isru_function.hpp

```diff
@@ -0,0 +1,135 @@
+/**
+ * @file isru_function.hpp
+ * @author Prince Gupta
+ *
+ * Definition and implementation of the ISRU function
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_ISRU_FUNCTION_HPP
+#define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_ISRU_FUNCTION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack{
+namespace ann /** Artifical Neural Network. */{
+
+/**
+ * The ISRU function, defined by
+ * 
+ * @f{eqnarray*}{
+ * f(x) &=& x / sqrt(1 + alpha* x ^ {2})
+ * f'(x) &=& cube(1 / sqrt(1 + alpha * x ^ {2}))
+ * @f
+ */
+
+class ISRUFunction
+{
+ public:
+  /**
+   * Computes the ISRU function
+   * 
+   * @param x Input data.
+   * @param alpha paramter, default value = 0.1
+   * @return f(x)
+   */
+  static double Fn(const double x, const double alpha = 0.1)
```

Comment:
We should implement this as a layer, similar to: https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/ann/layer/elu.hpp#L127 because there is no way, a user can use that function with a base layer and adjust the alpha parameter.